Client/Application/AppDelegate.swift

```diff
@@ -9,6 +9,7 @@ import SDWebImage
 import UIKit
 
 let LatestAppVersionProfileKey = "latestAppVersion"
+let NewPlacesAPIDefaultKey = "NewPlacesAPI"
```

Comment:
Would be good to move these to `PrefsKeys` file where all our UserDefaults Keys are stored